src/schema-manager/schema-manager-factory.ts

```diff
@@ -45,8 +62,15 @@ export class SchemaManagerFactory {
       return {
         http: {
           url: path,
-        } as HttpSchemaManagerOptions,
+        } as RequestSetup,
       };
+    }
+    if (/.js$/.test(path)) {
```

Comment:
I think  the `"schema": "./config.js"` shortcut is not needed and `"schema": { "http": { "fromScript": "./config.js" } }` is enough.

Reply:
Fixed.